wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1344,6 +1344,58 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+def get_keyboard_key_by_user_agent(user_agent):
+    """
+    Returns the standard keyboard keys based on user device
+    """
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+
+    if is_mac:
+        return {
+            "command/control": "⌘",
+            "option/alt": "⌥",
+            "escape": "Esc",
+            "shift": "Shift",
+            "enter/return": "Return",
+            "delete/backspace": "Delete",
+            "tab": "Tab",
+        }
+    else:
+        return {
+            "command/control": "Ctrl",
+            "option/alt": "Alt",
+            "escape": "Esc",
+            "shift": "Shift",
+            "enter/return": "Enter",
+            "delete/backspace": "Backspace",
+            "tab": "Tab",
+        }
+
+
+def get_wagtail_keyboard_actions(keyboard_keys):
```

Comment:
I recommend we just keep this logic within the flow of the existing code, it's a bit messy bring this out only.

I am sure we can abstract things better later if the need arises, pulling out the key label mapping is good enough for now.

Also, minor nitpick, when creating the variable we should keep the tense.

```py
    comments_enable = get_comments_enabled()
```

Should be

```py
    comments_enabled = get_comments_enabled()
```